roles/etcd/tasks/gen_certs_script.yml

```diff
@@ -18,18 +18,6 @@
   when: inventory_hostname == groups['etcd'][0]
   delegate_to: "{{ groups['etcd'][0] }}"
 
-- name: "Gen_certs | create etcd cert dir (on {{ groups['etcd'][0] }})"
-  file:
-    path: "{{ etcd_cert_dir }}"
-    group: "{{ etcd_cert_group }}"
-    state: directory
-    owner: kube
-    recurse: yes
-    mode: 0700
```

Comment:
The first "Gen_certs | create etcd cert dir" uses {{ etcd_cert_dir_mode }} instead of 0700, that seems good to be kept like this PR.

Reply:
yes.